### PR TITLE
Add ComboSecurityScheme with descriptions

### PR DESCRIPTION
**Pull request by FadySalama**
This PR adds ComboSecurityScheme definitions for TUM Philips Hue devices with descriptions in both English and German

### CODE REVIEW

File: events/2022.06.Online/TD/philips-hue/TDs/tum-daylight.td.jsonld [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        "https://www.w3.org/2022/wot/td/v1.1",
+        {
+            "@language": "en"
+        }
+    ],
+    "title": "DaylightSensor",
+    "security": [
+        "combo_sc"
+    ],
+    "security": "combo_sc",
```
Comment (egekorkan) [inline events/2022.06.Online/TD/philips-hue/TDs/tum-daylight.td.jsonld:13]: Security is used twice

File: events/2022.06.Online/TD/philips-hue/TDs/tum-daylight.td.jsonld [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    "description": "The name of the device manufacturer."
+                },
+                "swversion": {
+                    "type": "string",
+                    "description": "This parameter uniquely identifies the software version running in the hardware."
+                }
+            },
+            "forms": [
+                {
+                    "href": "https://esiremotelab.esi.ei.tum.de:8081/LabLocal/api/17AvYQeLM2T8Z3r0e3BiEvb9Qdwt36yU2BudTxrn/sensors/1",
```
Comment (egekorkan) [inline events/2022.06.Online/TD/philips-hue/TDs/tum-daylight.td.jsonld:106]: This should be updated to reflect apikey security scheme

**Review by egekorkan — CHANGES_REQUESTED** [review]
My comments apply to all changed tds

**Comment by FadySalama** [comment]
> Lookd good

@egekorkan I found the error that causes the test to fail. (See: [here](https://github.com/w3c/wot-thing-description/issues/1557))

**Pull request merged**